pkg/kubelet/eviction/helpers.go

```diff
@@ -204,14 +255,52 @@ func memoryUsage(memStats *statsapi.MemoryStats) *resource.Quantity {
 	return resource.NewQuantity(usage, resource.BinarySI)
 }
 
-// podUsage aggregates usage of compute resources.
-// it supports the following memory and disk.
-func podUsage(podStats statsapi.PodStats) (api.ResourceList, error) {
+// localVolumeNames returns the set of volumes for the pod that are local
```

Comment:
i agree long term.  i will add a // TODO for follow-on.

for this PR, i will just ensure that empty dir is only included when medium is not memory.